### PR TITLE
Feature/11907 product page layout

### PR DESCRIPTION
**Pull request by erochOnGit**

### CODE REVIEW

File: src/web/theme/pages/TryItPipe/TryItPipe.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React from "react";
+
+const TryItPipe = () => {
+  return <div>Work In Progress</div>;
+};
+
+TryItPipe.prototype = {};
```
Comment (hihuz) [inline src/web/theme/pages/TryItPipe/TryItPipe.js]: It is a WIP so this is not critical but I think you meant "propTypes" here :laughing:

File: src/web/Router.js [hunk 2]
```diff
@@ -9,6 +9,7 @@ import Footer from "./theme/modules/Footer";
 import Home from "./theme/pages/Home";
 import Product from "./theme/pages/Product";
 import Checkout from "./theme/pages/Checkout";
+import TryItPipe from "./theme/pages/TryItPipe";
```
Comment (hihuz) [inline src/web/Router.js]: I don't understand the term "Pipe" in the name of this, care to explain ? :smiley:

**Review by hihuz — APPROVED** [review]
Nice job :+1:

File: src/web/theme/ui/atoms/Button/Button.js [hunk 3]
```diff
@@ -24,7 +24,7 @@ Button.propTypes = {
   onClick: PropTypes.func,
   children: PropTypes.node.isRequired,
   status: PropTypes.string,
-  type: PropTypes.oneOf(["default", "invisible", "full-size"])
+  type: PropTypes.oneOf(["default", "invisible", "full-size", "add-to-cart"])
```
Comment (milkae) [inline src/web/theme/ui/atoms/Button/Button.js]: That prop is so weird, the type is meant to be "submit", "button" or "reset". And it is necessary for handling form with many buttons. Having such value for this element property is a nonsense.
Is that part of fc-lite ?

**Review by milkae — APPROVED** [review]
Clean ! :star:

File: src/web/theme/pages/TryItPipe/TryItPipe.js (hunk 1 above)
Comment (erochOnGit) [reply src/web/theme/pages/TryItPipe/TryItPipe.js]: X) yep

File: src/web/Router.js (hunk 2 above)
Comment (erochOnGit) [reply src/web/Router.js]: i thought about a "tunnel" .. but i'll just remove it it's not necessary

**Pull request merged**